### PR TITLE
Changes related to #24958

### PR DESCRIPTION
**Pull request by kamlekar**
cc @XhmikosR 

Related Pull request: https://github.com/twbs/bootstrap/pull/24511

Fixes: #24958

### CODE REVIEW

**Comment by XhmikosR** [comment]
IMO we should remove the whole object and use the unprefixed property
directly.

**Comment by kamlekar** [comment]
@XhmikosR  Yes, that would be far better. Let me try it out. Please feel free to close if someone do it already.

**Comment by kamlekar** [comment]
@XhmikosR done.

File: js/src/util.js [hunk 1]
```diff
@@ -50,11 +45,9 @@ const Util = (($) => {
 
     const el = document.createElement('bootstrap')
```
Comment (Johann-S) [inline js/src/util.js]: they are no needs now to create an element here just return : 
```js
return {
  end: 'transitionend'
}
```

that's all

**Review by Johann-S — CHANGES_REQUESTED** [review]

File: js/src/util.js (hunk 1 above)
Comment (kamlekar) [reply js/src/util.js]: @Johann-S done and committed.

**Comment by Johann-S** [comment]
You have to fix that : https://travis-ci.org/twbs/bootstrap/jobs/312883636#L1063

**Review by Johann-S — APPROVED** [review]

**Pull request merged**